server/src/ScriptRunner/Scripts/index/BLK_test.sql

```diff
@@ -0,0 +1 @@
+SELECT count(id) from public.userz;
```

Comment:
why userz with z?

Reply:
this is to simulate an invalid change - if you'll run this script it'll return ERROR - ill remove all scripts once the PR will be completed